charts/seqr/templates/environment.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
 kind: ConfigMap
+apiVersion: v1
```

Comment:
I can't find this written down anywhere as a spec or lint rule, and I don't think it has any effect, but every example I've come across in the k8s docs and in other charts has the apiVersion on the first line, e.g. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment

At the risk of being too persnickety, I'd say we should move this back, so that it matches our other charts, as well as manifests generated by tools like `helm create`

Reply:
Cool, no worries from me here.  I actually toggled this for consistency with the majority of the other templates.  I believe `service.yaml`, `ingress.yaml`, and `deployment.yaml` are all `apiVersion` second, so I went with that standard.  I also feel like I've seen more examples with `apiVersion` first though, so that makes sense to me!